tasks/set_sql_logins.json

```diff
@@ -0,0 +1,28 @@
+{
+  "puppet_task_version": 1,
+  "supports_noop": false,
```

Comment:
Would be neat if `noop` was supported?

Reply:
Adding it. I agree this would be useful.